tools/flaky_test_monitor/level3/flaky-test-monitor.properties

```diff
@@ -0,0 +1,21 @@
+# property file for Flaky Test Monitor
+
+##############################
+#####   LEVEL 3  #############
+##############################
+
+# includes tests that have a failure rate of at least the specified percentage
+# e.g. 0.40 == 40%, 0.75 == 75%
+failures_threshold_percent=0
+
+# max number of test failures to return in slice that's at least `failures_threshold_percent`
+# does not impact total number of tests with failure rate at least `failures_threshold_percent` which will be reported separately
+failures_slice_max=10
+
+# include tests that are at least this many seconds long
+# this is meant to filter out the tests that aren't as long running
+duration_threshold_seconds=0
```

Comment:
do we want 0 as a default here?

Reply:
Good point. I increased it to 0.05 seconds

https://github.com/onflow/flow-go/pull/1793/commits/7ca42b8a930036d6e038e08ad342650a254e482a